apps/total-typescript/package.json

```diff
@@ -4,7 +4,7 @@
   "private": true,
   "scripts": {
     "dev": "next dev -p 3016",
-    "dev:setup": "vercel env pull .env.local",
+    "dev:setup": "vercel env pull .env.local && sed -i.bak '/^VERCEL/d' .env.local",
```

Comment:
I think this should migrate to a bash script, or node if we want it to work outside of *nix